extensions/big-data-cluster/src/wizards/create-cluster/resourceStrings.ts

```diff
@@ -0,0 +1,57 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+'use strict';
+
+import * as nls from 'vscode-nls';
+
+const localize = nls.loadMessageBundle();
```

Comment:
The pattern most of our code has moved to is to declare strings inline in the source file they're used in. Can we do this? Having a separate localize file has no benefit and can make it hard to quickly update things